recipe/meta.yaml

```diff
@@ -10,21 +10,22 @@ source:
   sha256: 0a2651dd2b9d7662c70d12ada5c290abcf57373b9633515fe4baa9f62566086f
 
 build:
-  noarch: python
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<36]
```

Comment:
You can remove this line as conda-forge doesn't support anything older than Python 3.6 anymore.

Otherwise looks good to me!

Reply:
You are right. I removed the constraint and will merge after the test passes. Thanks